app/javascript/components/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { Routes, Route } from 'react-router';
+import { useSelector } from 'react-redux';
+import { FiMenu } from 'react-icons/fi';
+import Main from './auth/Main';
+import Navbar from './Navbar/Navbar';
+import AddItem from './AddItem';
+import Reservation from './Reservation';
+import Details from './Details';
+import DeleteItem from './DeleteItem';
+import Registration from './auth/registration';
+import Login from './auth/Login';
+import ReserveForm from './ReserveForm';
+import Mainpage from './Mainpage';
+
+const App = () => {
```

Comment:
- [ ] According to the project requirements:

![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/0b5bdc74-6e6e-4e44-a1bf-19d3a07fb694)

Your design implementation seems to be questionable:

Your Implementation | Given Design
:---:|:---:
![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/67dbcefa-af0b-48e7-8a43-a372dfe17240) | ![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/58532897-9146-4116-8ef0-d13109e2a162)

Kindly, add more styling/touch ups so the implementation looks more like the given design.  More specifically, the font stylings, and the styling of the images which seem to be cut in the top.

### Tips and extra help

I was able to make the images a little more like the design by doing:

![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/7d99bff7-3a91-4d54-b88b-f9b3d204e29c)

#### Result:

![image](https://github.com/Kaghenimbale/car_collection/assets/57421823/1c47b2c1-c925-4699-9b5f-5497cf7867ff)


This is just an example meant to be a starting point for you.  Please try your best to make them more like the design. 💪